src/ngAnimate/animateCss.js

```diff
@@ -623,6 +623,7 @@ var $AnimateCssProvider = ['$animateProvider', function($animateProvider) {
 
       if (options.delay != null) {
         var delayStyle = parseFloat(options.delay);
+        maxDelay = Math.max(options.delay, 0);
```

Comment:
`delayStyle` is just `parseFloat(options.delay)`.

But thinking about it again, neither option will work as expected in case `options.delay === true` (which is a valid option according to the docs).
From a superficial look, we might need to make something like on [animateCss.js#L849-L850][1] and animateCss.js#L863-L867][2].

[1]: https://github.com/Narretz/angular.js/blob/fix-nganimate-css-13355/src/ngAnimate/animateCss.js#L849-L850
[2]: https://github.com/Narretz/angular.js/blob/fix-nganimate-css-13355/src/ngAnimate/animateCss.js#L863-L867